cookbook/core/containerization/Dockerfile_preprocessing

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /root
+ENV VENV /opt/venv
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV PYTHONPATH /root
+
+# Install skllearn
+RUN pip3 install scikit-learn
+
+# Install Python dependencies
+COPY containerization/requirements.txt /root/.
+RUN pip install -r /root/requirements.txt
+
+# Copy the makefile targets to expose on the container. This makes it easier to register.
+COPY in_container.mk /root/Makefile
+COPY containerization/sandbox.config /root
+
+# Copy the actual code
+COPY containerization/ /root/containerization/
```

Comment:
In the dockerfile, you would also have to include https://github.com/flyteorg/flytesnacks/blob/0734a90b4986ff91ae8a5a48e68fc44795c38c7d/cookbook/core/Dockerfile#L13-L25 and https://github.com/flyteorg/flytesnacks/blob/0734a90b4986ff91ae8a5a48e68fc44795c38c7d/cookbook/core/Dockerfile#L38-L41.